sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -687,6 +692,10 @@ Exceptions:
 * Throws synchronously with error code `invalid` if the any queue in
   `recordingQueues` is already recording to a different graph.
 
+* Throws synchronously with error code `invalid` if any of `recordingQueues`
```

Comment:
GitHub won't let me comment on the relevant line so doing it here instead...

Think we might also need an error specified on `update()` if trying to update to a modifiable graph with a different context. Or make a comment on my "topologically identical" PR to include a mention of context.

Reply:
I've added an error for this in the specification of `update()`.